### DISCUSSION TITLE
Contributions not showing

### DISCUSSION THREAD

**Discussion by sketchbuch** [community/community#76905] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

I was working on a PR for several weeks, an after merging to my repo's main branch the contributions are not showing the commits that I made.

All the commits were made with the correct email address, git is setup on my system corrctly to use this address and other commits show up and have for years using this address, but none of the commits from the merged branch show on my profile overview. It has been nearly 48 hours now.

Is there anything I can do about this?

Here are the commits:

https://github.com/sketchbuch/vsc-workspace-sidebar/pull/93/commits

**Comment by guspan-tanadi** [comment, marked as answer]
Likely the commits are [squashed](https://docs.github.com/en/repositories/configuring-branches-and-merges-in-your-repository/configuring-pull-request-merges/about-merge-methods-on-github#squashing-your-merge-commits) into one commit. For example when merging through `feat/61-multiple-root-folders` to `master`. Moreover, when counting as of November 2023 there are available [3 commits](https://github.com/sketchbuch/vsc-workspace-sidebar/commits/master) on `master` same number as the profile overview.

**Comment by sketchbuch** [comment]
OK; thanks for the clarification... I did squash. I remember now. Thank you!
